google/api_core/bidi.py

```diff
@@ -276,7 +276,11 @@ def open(self):
         request_generator = _RequestQueueGenerator(
             self._request_queue, initial_request=self._initial_request
         )
-        call = self._start_rpc(iter(request_generator), metadata=self._rpc_metadata)
+        try:
```

Comment:
In examining this file, it seems to me that `_RequestQueueGenerator._is_active`  (lines 91 et seq) should be `def`d as `return self.call is not None and self.call.is_active()`. The way it's currently written it will return `True` when `self.call == None`

Reply:
Fixed in e120a0cdb589d390848b0711ff21c9ff4aab26a9